scm-ui/ui-components/src/repos/RepositoryEntry.tsx

```diff
@@ -152,7 +152,7 @@ class RepositoryEntry extends React.Component<Props, State> {
   createFooterRight = (repository: Repository, baseDate?: DateProp) => {
     return (
       <small className="level-item">
-        <DateFromNow baseDate={baseDate} date={repository.creationDate} />
+        <DateFromNow baseDate={baseDate} date={repository.lastModified} />
```

Comment:
Newly created repository do not have a last modified date. Does it make sense to use the creation date for repositories without last modified date?

```suggestion
        <DateFromNow baseDate={baseDate} date={repository.lastModified || repository.creationDate} />
```

Reply:
Good point. We should do this.